build-scripts/gulp/translations.js

```diff
@@ -99,18 +98,16 @@ function recursiveEmpty(data) {
  * @link https://docs.lokalise.co/article/KO5SZWLLsy-key-referencing
  */
 const re_key_reference = /\[%key:([^%]+)%\]/;
-function lokalise_transform(data, original) {
+function lokaliseTransform(data, original, file) {
```

Comment:
We so much need to re-do this file in a non-stream way so we can actually see what's going on .